.github/workflows/publish.yml

```diff
@@ -96,13 +96,5 @@ jobs:
         run: |
           demo_current_version=$(.github/script/get-demo-browser-current-version)
           echo "Current demo version: " $demo_current_version
-          echo "Current released npm version:" ${{ steps.npm_version.outputs.npm_version }}
-          if [[ *$demo_current_version* != "^#${{ steps.npm_version.outputs.npm_version }}" ]];
-          then
-            echo "Demo npm version is not the same as the current released version"
-            exit 1
-          else
-            npm run build --app=meetingReadinessChecker
```

Comment:
we dont need this?

Reply:
No. After looking at the workflow it turns out that this if statement will never be true.

Reply:
This if statement's string comparison was the reason why the script was failing.

Reply:
Why we need to remove this:
>npm run build --app=meetingReadinessChecker

Reply:
The deploy script already build it